doc/manual/queryformat

```diff
@@ -87,21 +86,21 @@ for passing information to any program that's line oriented (such as
 grep or awk). If you try the obvious,
 
 \verbatim
-    rpm -q --queryformat "[%{NAME} %{FILENAMES}\n]" cdp
+    rpm -q --queryformat "[%{NAME} %{FILENAMES}\n]" systemd
 \endverbatim
 
-If you try this, you'll see RPM complain about an "array iterator used
-with different sized arrays". Internally, all items in RPM are actually
-arrays, so the NAME is a string array containing one element. When you
-tell RPM to iterate over the NAME and FILENAMES elements, RPM notices
-the two tags have different numbers of elements and complains.
+you'll see RPM complain about an "array iterator used with different
+sized arrays". Internally, all items in RPM are actually arrays, so the
+NAME is a string array containing one element. When you tell RPM to iterate
+over the NAME and FILENAMES elements, RPM notices the two tags have
+different numbers of elements and complains.
 
 To make this work properly, you need to tell RPM to always print the first
 item in the NAME element. You do this by placing a '=' before the tag
 name, like this:
 
 \verbatim
-    rpm -q --queryformat "[%{=NAME} %{FILENAMES}\n]" cdp
+    rpm -q --queryformat "[%{=NAME} %{FILENAMES}\n]" systemd
```

Comment:
If you change this, then you also need to change the output sample below.

We can't list the entire systemd package contents there so dunno if it's really worth it. Looking closer, there are several other now extinct package names mentioned in the document too, IF we want to update then maybe we should update them all. Or just leave them as historical reference for future generations to gape at...